roles/dev/tasks/os_agnostic_tasks.yml

```diff
@@ -1,11 +1,11 @@
 - name: download Golang v1.6
   get_url:
     validate_certs: "{{ validate_certs }}"
-    url: https://storage.googleapis.com/golang/go1.6.linux-amd64.tar.gz
+    url: https://storage.googleapis.com/golang/go1.6.2.linux-amd64.tar.gz
     dest: /tmp/go1.6.linux-amd64.tar.gz
```

Comment:
the `dest` filename needs update as well. The current sanity failure is related to this.

May be we can make the go version a variable and use that everywhere, just like we do for some other roles like contiv_network, contiv_storage etc.